tests-perf/locust/locust-notifications.py

```diff
@@ -12,13 +12,40 @@ class NotifyApiUser(HttpUser):
     wait_time = constant_pacing(60)
     host = "https://api.staging.notification.cdssandbox.xyz"
 
-    @task
-    def send_notifications(self):
-        headers = {"Authorization": AUTH_HEADER}
-        json = {
+    def __init__(self, *args, **kwargs):
+        super(NotifyApiUser, self).__init__(*args, **kwargs)
+
+        self.headers = {"Authorization": AUTH_HEADER}
+        self.json = {
             "email_address": "success@simulator.amazonses.com",
-            "template_id": "9c17633c-126a-4ad3-ad2f-b14c3a85314a",
-            "personalisation": {"colour": "Fulvous"},
+            "template_id": "5ebee3b7-63c0-4052-a8cb-387b818df627",
+            "personalisation": {},
+        }
+
+    @task(1)
+    def send_email_notifications(self):
+        self.client.post("/v2/notifications/email", json=self.json, headers=self.headers)
+
+    @task(2)
+    def send_email_with_attachment_notifications(self):
+        self.json["personalisation"] = {
+            "application_file": {
+                "file": "Q29udGVudCBvZiBBdHRhY2hlZCBmaWxl",
+                "filename": "attached_file.txt",
+                "sending_method": "attach",
+            }
+        }
+
+        self.client.post("/v2/notifications/email", json=self.json, headers=self.headers)
+
+    @task(4)
+    def send_email_with_link_notifications(self):
+        self.json["personalisation"] = {
+            "var": {
```

Comment:
This `var` would fail here as the template would need it. `application_file` seems like the proper variable name to use.

Reply:
Ok will trying changing bar for application_file.. Cheers